src/hotspot/share/opto/type.cpp

```diff
@@ -3301,12 +3308,26 @@ bool TypePtr::InterfaceSet::eq(const InterfaceSet& other) const {
   return true;
 }
 
-int TypePtr::InterfaceSet::hash() const {
-  if (_hash_computed) {
-    return _hash;
+bool TypePtr::InterfaceSet::eq(ciInstanceKlass* k, InterfaceHandling interface_handling) const {
```

Comment:
Why not remove the interface_handling parameter? It doesn't seem useful.

Reply:
Right, I'll remove it.